wolfcrypt/src/misc.c

```diff
@@ -101,6 +113,8 @@ STATIC INLINE word32 ByteReverseWord32(word32 value)
     return (word32)__REV(value);
 #elif defined(KEIL_INTRINSICS)
     return (word32)__rev(value);
+#elif defined(__GNUC_PREREQ) && __GNUC_PREREQ(4, 3)
+    return (word32)__builtin_bswap32(value);
```

Comment:
Did you benchmark this change?  Is is faster than fast rotate for example?

Reply:
I did not benchmark this, but will. I expect on some platforms it will be faster.